fapolicy_analyzer/ui/actions.py

```diff
@@ -62,6 +81,12 @@ class Notification(NamedTuple):
     type: NotificationType
 
 
+class DaemonState(NamedTuple):
+    error: str
+    status: bool
+    handle: Handle
```

Comment:
I don't see anywhere this `handle` is being used.  Also it shouldn't really be exposed in the application state.  If something what's to interact with the ref being held by the redux feature, they should be doing through some action.

Reply:
Ok. I was thinking that if it's not set i.e. None, then the ui layer could infer it's not installed and dim out all backend operation elements. But your point about not exposing it to the UI layer is a good one, Should we have just a boolean?

Reply:
What if you make the `status` 3 states `(None, True, False)` where `None` would indicate that its not installed or the status cannot be pulled.

Reply:
Currently revisiting DaemonState...

Reply:
Made status an enum TRUE, FALSE, UNKNOWN.